### ISSUE TITLE
Smilie select is not working

### ISSUE THREAD

**Issue by euantorano** [mybb/mybb#3554]
The smilie selector box when creating a thread is not working. The following error is logged in the console:

```
TypeError: null is not an object (evaluating 'MyBBEditor.insertText')
```

`MyBBEditor` is indeed null.

**Comment by euantorano** [comment]
It looks like this may be due to an issue in SCEditor with jQuery 3.0. I'm also getting the following JS error:

```
jQuery.Deferred exception: a.indexOf is not a function. (In 'a.indexOf(" ")', 'a.indexOf' is undefined) (2)
load — jquery.js:3489
K — jquery.sceditor.bbcode.min.js:2:1922
o — jquery.sceditor.bbcode.min.js:2:24227
(anonymous function) — jquery.sceditor.bbcode.min.js:2:826
each — jquery.js:143
sceditor — jquery.sceditor.bbcode.min.js:2:665
(anonymous function) — newthread.php:432
j — jquery.js:1333
(anonymous function) — jquery.js:1340
```

**Comment by dvz** [comment]
https://github.com/mybb/mybb/pull/3555#issuecomment-462245895

**Comment by dvz** [comment]
Resolved by #3574
